Content.Server/Nyanotrasen/Shipyard/Systems/ShipyardSystem.cs

```diff
@@ -90,10 +90,14 @@ public bool TryPurchaseShuttle(EntityUid stationUid, string shuttlePath, [NotNul
                 return false;
             }
 
+            /// nyano change:
+            /// Allow Salvage Expedition Computers to work on Shipyard shuttles by tracking the owning Station.
+            _station.AddGridToStation(stationUid, (EntityUid) shuttleGrid);
+            /// end of change
+
```

Comment:
The documentation is appreciated, but this file is in one of our own Nyanotrasen directories, so the block format described in the [contributing guidelines](https://github.com/Nyanotrasen/Nyanotrasen/wiki/Contributing-Guidelines#changing-upstream-code) is unnecessary here.

Triple slashes also indicate [documentation comments](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments), which isn't quite what you'd use here. Just a regular double slash comment explaining the logic is fine.

Reply:
Ah, yeah, I thought that was strange. I was just copy-pasting Rane's format [here](https://canary.discord.com/channels/1097912084345405490/1098374574690807911/1108040050857492601) for consistency's sake. I'll fix that up next chance I get.